src/Windows/Perspex.Win32/Interop/UnmanagedMethods.cs

```diff
@@ -751,6 +754,8 @@ internal class FileOpenDialogRCW { }
         [DllImport("comdlg32.dll")]
         public static extern int CommDlgExtendedError();
 
+        public static bool ShCoreAvailable => LoadLibrary("shcore.dll") != IntPtr.Zero;
```

Comment:
At first step I have been implemented caching :) But after that I noticed that shcore.dll methods called one time at application start. Ok, if you argue I'll implement caching.